XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
I've migrated into unified scheme. Generally I checked https://github.com/mxcl/PromiseKit settings to do this.

The main thing that can be wrong is that I had to remove `SwiftFoundation.framework` from "Linked Frameworks and Libraries" since it doesn't compile for iOS platform. I checked that there's a setting "Link Frameworks Automatically", but I'm not sure that it works properly.

Could anyone please provide any suggestions?